src/main/resources/application.yaml

```diff
@@ -243,6 +243,11 @@ exit-survey:
   applicant-link: https://www.smartsurvey.co.uk/s/CivilDamages_ExitSurvey_Claimant/
   respondent-link: https://www.smartsurvey.co.uk/s/CivilDamages_ExitSurvey_Defendant/
 
+polling:
+  event:
+    emitter:
+      enabled: ${POLLING_EVENT_EMITTER_ENABLED:false}
+
```

Comment:
im confused as to what the change is here? you just moved a few lines above?

Reply:
Previously this was for only idam profile and not for default. Hence it was not being picked up in prod. So we have to move in default profile.